products/samsungmobile.md

```diff
@@ -898,181 +898,181 @@ releases:
-    eol: false
+    support: false
+    eol: true
     link: https://doc.samsungmobile.com/SM-N970F/XEF/doc.html
 
 -   releaseCycle: "Galaxy Tab S6"
     releaseDate: 2019-08-01
-    support: true
-    eol: false
+    support: false
+    eol: true
     link: https://doc.samsungmobile.com/SM-T860/XAR/doc.html
 
 -   releaseCycle: "Galaxy Note10+ 5G"
     releaseDate: 2019-08-01 # Approximate to the month and year.
-    support: true
-    eol: false
+    support: false
+    eol: true
     link: https://doc.samsungmobile.com/SM-N976B/SFR/doc.html
 
 -   releaseCycle: "Galaxy Note10 5G"
     releaseDate: 2019-08-01 # Approximate to the month and year.
-    support: true
-    eol: false
+    support: false
+    eol: true
     link: https://doc.samsungmobile.com/sm-n971n/koo/doc.html
 
 -   releaseCycle: "Galaxy Tab A 8.0 (2019)"
     releaseDate: 2019-07-01
     support: false
-    eol: false
+    eol: true
     link: https://doc.samsungmobile.com/SM-T290/008916190830/mlt.html
 
 -   releaseCycle: "Galaxy Xcover 4s"
     releaseDate: 2019-07-01 # Approximate to the month and year.
     support: 2022-10-01 # Approximate from here: https://9to5google.com/2022/10/04/samsung-android-12-update-rollout/
-    eol: false
+    eol: true
     link: https://doc.samsungmobile.com/SM-G398FN/ATO/doc.html
 
 -   releaseCycle: "Galaxy A60"
     releaseDate: 2019-06-01
     support: false
-    eol: false
+    eol: true
     link: https://doc.samsungmobile.com/SM-A6060/TGY/doc.html
 
 -   releaseCycle: "Galaxy M40"
     releaseDate: 2019-06-01 # Approximate to the month and year.
-    support: true
-    eol: false
+    support: false
+    eol: true
     link: https://doc.samsungmobile.com/SM-M405F/INS/doc.html
 
 -   releaseCycle: "Galaxy A80"
     releaseDate: 2019-05-01
     support: false
-    eol: false
+    eol: true
     link: https://doc.samsungmobile.com/SM-A805F/XEH/doc.html
 
 -   releaseCycle: "Galaxy A70"
     releaseDate: 2019-05-01
     support: false
-    eol: false
+    eol: true
     link: https://doc.samsungmobile.com/SM-A705F/XID/doc.html
 
 -   releaseCycle: "Galaxy A20e"
     releaseDate: 2019-05-01
     support: false
-    eol: false
+    eol: true
     link: https://doc.samsungmobile.com/SM-A202F/DBT/doc.html
 
 -   releaseCycle: "Galaxy A20"
     releaseDate: 2019-04-05
     support: false
-    eol: false
+    eol: true
     link: https://doc.samsungmobile.com/SM-A205F/AFR/doc.html
 
 -   releaseCycle: "Galaxy A40"
     releaseDate: 2019-04-01
     support: 2022-10-01 # Approximate from here: https://9to5google.com/2022/10/04/samsung-android-12-update-rollout/
-    eol: false
+    eol: true
     link: https://doc.samsungmobile.com/SM-A405FN/XEH/doc.html
 
 -   releaseCycle: "Galaxy Watch Active"
     releaseDate: 2019-04-01 # Approximate to the month and year.
-    support: true
-    eol: false
+    support: false
+    eol: true
     link: https://doc.samsungmobile.com/sm-r500/xar/doc.html
 
 -   releaseCycle: "Galaxy Tab A 8.0 & S Pen (2019)"
```

Comment:
Outside the scope of this PR, but lLooks like a duplicate of "Galaxy Tab A 8.0 with S Pen (2019)"

Reply:
#4295.